MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -105,7 +105,7 @@
 
     <!-- This string is used as the title of the pop-ups dialog used to remove sites from the currently
          allowed pop-up sites. -->
-    <string name="settings_privacy_policy_popups_title">Pop-ups permissions</string>
+    <string name="settings_privacy_policy_popups_title">Exceptions for Pop-Ups</string>
```

Comment:
@keianhzo
Changes to existing strings require new string IDs.

Reply:
I've updated the key.